src/stories/Library/event-header/EventHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from "react";
+import { Tag } from "../tag/Tag";
+import { Button } from "../Buttons/button/Button";
+
+type EventHeaderProps = {
+  title: string;
+  date: string;
+};
+
+const EventHeader: FC<EventHeaderProps> = ({ title, date }) => {
+  return (
+    <header className="event-header">
+      <section className="event-header__content">
+        <div className="event-header__extra-padding">
+          <Tag hasBackground className="event-header__tag">
+            UDSTILLING
+          </Tag>
+          <time className="event-header__date">{date}</time>
+          <h1 className="event-header__title">{title}</h1>
+        </div>
+        <Button
+          classNames="event-header__button"
+          label="Køb billet"
+          buttonType="none"
+          disabled={false}
+          collapsible={false}
+          size="large"
+          variant="filled"
+        />
+      </section>
+      <section className="event-header__visual">
```

Comment:
I suggest replacing `<section>` with [`<figure>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure).

There does not seem to be a clean cut solution on how to implement credits but [people on Stackoverflow has a nice discussion about the subject](https://stackoverflow.com/questions/57899136/how-to-implement-a-caption-and-photographer-credit-with-figure).

Reply:
Consider implementing a separate block for credited images.

In general this setup with an image with credits below seems to be used several places outside of the event header. I think it is OK not to do it here, now, but we may need to refactor it in the future.